CHANGES.txt

```diff
@@ -4,12 +4,20 @@ CHANGES
 0.18 (unreleased)
 =================
 
+* **Removed**: ``morepath.body_model_predicate`` is removed from the
+  Morepath API together with the ``morepath.App.load_json`` directive
+  and the ``morepath.request.body_obj`` property.
+  The ``load_json`` infrastructure has been moved to the `more.body_model`_
```

Comment:
I'd swap these so make it explicit what to do first. "If you use the ``load_json`` directive, this functionality has been moved to a separate package ``more.body_model``. Use this package by subclassing from its app class (XXX proper details) if you use the ``load_json`` directive."